lightly/api/api_workflow_datasources.py

```diff
@@ -101,6 +101,8 @@ def download_raw_samples(
         from_: int = 0,
         to: Optional[int] = None,
         relevant_filenames_file_name: Optional[str] = None,
+        run_id: Optional[str] = None,
+        relevant_filenames_artefact_id: Optional[str] = None,
```

Comment:
We use "artifact" everywhere else so please change this for consistency.

Reply:
Updated